docs/en_US/Compression/Tutorial.rst

```diff
@@ -0,0 +1,190 @@
+Tutorial
+========
+
+.. contents::
+
+In this tutorial, we will explain more detailed usage about the model compression in NNI. 
+
+Setup compression goal
+----------------------
+
+Specify the configuration
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Users can specify the configuration (i.e., ``config_list``\ ) for a compression algorithm. For example, when compressing a model, users may want to specify the sparsity ratio, to specify different ratios for different types of operations, to exclude certain types of operations, or to compress only a certain types of operations. For users to express these kinds of requirements, we define a configuration specification. It can be seen as a python ``list`` object, where each element is a ``dict`` object. 
+
+The ``dict``\ s in the ``list`` are applied one by one, that is, the configurations in latter ``dict`` will overwrite the configurations in former ones on the operations that are within the scope of both of them. 
+
+There are different keys in a ``dict``. Some of them are common keys supported by all the compression algorithms:
+
+* **op_types**\ : This is to specify what types of operations to be compressed. 'default' means following the algorithm's default setting.
+* **op_names**\ : This is to specify by name what operations to be compressed. If this field is omitted, operations will not be filtered by it.
+* **exclude**\ : Default is False. If this field is True, it means the operations with specified types and names will be excluded from the compression.
+
+Some other keys are often specific to a certain algorithms, users can refer to `pruning algorithms <./Pruner.rst>`__ and `quantization algorithms <./Quantizer.rst>`__ for the keys allowed by each algorithm.
```

Comment:
a certain algorithms -> a certain algorithm?

Reply:
fix it